torch/nn/utils/rnn.py

```diff
@@ -428,14 +436,19 @@ def unpad_sequence(
         padded_sequences (Tensor): padded sequences.
         lengths (Tensor): length of original (unpadded) sequences.
         batch_first (bool, optional): whether batch dimension first or not. Default: False.
+        inplace (bool, optional): If ``True``, ``padded_sequences`` will be transposed
+            into ``B x T x *`` format in-place when ``batch_first`` is ``False``. Default: ``False``.
 
     Returns:
         a list of :class:`Tensor` objects
     """
     unpadded_sequences = []
 
     if not batch_first:
-        padded_sequences.transpose_(0, 1)
+        if inplace:
```

Comment:
If we are preserving the current behavior (not breaking BC), I think `inplace` should be `True` by default.